coderd/notifications/notifications_test.go

```diff
@@ -1610,3 +1611,183 @@ func (n *acquireSignalingInterceptor) AcquireNotificationMessages(ctx context.Co
 	n.acquiredChan <- struct{}{}
 	return messages, err
 }
+
+func TestNotificationTemplates_GoldenWithCustomAppearance(t *testing.T) {
+	t.Parallel()
+
+	if !dbtestutil.WillUsePostgres() {
+		t.Skip("This test requires postgres; it relies on the notification templates added by migrations in the database")
+	}
+
+	const (
+		username = "bob"
+		password = "🤫"
+
+		hello = "localhost"
+
+		from = "system@coder.com"
+		hint = "run \"DB=ci make update-golden-files\" and commit the changes"
+	)
+
+	payload := types.MessagePayload{
+		Labels: map[string]string{
+			"name":      "bobby-workspace",
+			"reason":    "autodeleted due to dormancy",
+			"initiator": "autobuild",
+		},
+	}
+
+	// Spin up the DB
+	db, logger, user := func() (database.Store, *slog.Logger, *codersdk.User) {
+		adminClient, _, api := coderdtest.NewWithAPI(t, nil)
+		firstUser := coderdtest.CreateFirstUser(t, adminClient)
+
+		_, user := coderdtest.CreateAnotherUserMutators(
+			t,
+			adminClient,
+			firstUser.OrganizationID,
+			[]rbac.RoleIdentifier{rbac.RoleUserAdmin()},
+			func(r *codersdk.CreateUserRequestWithOrgs) {
+				r.Username = "bobby"
+				r.Email = "bobby@coder.com"
+				r.Name = "Bobby"
+			},
+		)
+		return api.Database, &api.Logger, &user
+	}()
+
+	// nolint:gocritic // Unit test.
+	ctx := dbauthz.AsSystemRestricted(testutil.Context(t, testutil.WaitSuperLong))
+
+	err := db.UpsertApplicationName(ctx, "CustomApplication")
```

Comment:
nit: Custom Application (space bar)

PS. The more natural golden files look, the better for testing coderd logic.